frontend/amundsen_application/static/js/config/config-default.ts

```diff
@@ -241,6 +203,42 @@ const configDefault: AppConfig = {
       },
       notices: {},
     },
+    [ResourceType.feature]: {
+      displayName: 'ML Features',
+      supportedSources: {
+        hive: {
+          displayName: 'Hive',
+          iconClass: 'icon-hive',
+        },
+      },
+      filterCategories: [
+        {
+          categoryId: 'entity',
+          displayName: 'Entity',
+          helpText: 'Enter exact tag name or a regex wildcard pattern',
```

Comment:
`helpText` for this and others should be updated. `tag` reference should only be on the tag filter.